### DISCUSSION TITLE
Argon song progress bar should discriminate the current position

### DISCUSSION THREAD

**Discussion by smoogipoo** [ppy/osu#23856] [Ideas] [answered]
A good map to test is https://osu.ppy.sh/beatmapsets/1002234#osu/2097831.

At high background dim levels (e.g. 70%), the progress bar looks reasonable:
![image](https://github.com/ppy/osu/assets/1329837/19c52f10-f1de-457d-91f9-fb283eba692f)

At lower dim levels (e.g. 20%), it's hard to discern where the current position is:
![image](https://github.com/ppy/osu/assets/1329837/3bca5dc9-6cab-41fa-bd34-d2fd6e7aa09f)

A few ideas...

1: A cap:
![image](https://github.com/ppy/osu/assets/1329837/0de2ba30-0cea-48e7-81a5-b9364d285e83)

2: Coloured bar:
![image](https://github.com/ppy/osu/assets/1329837/c84ef22f-114a-493a-842b-c4cc49e5b50f)
Don't look at the particular hue.

3: Coloured tiers:
![image](https://github.com/ppy/osu/assets/1329837/9461b8f7-de95-4e9d-9bcd-55e2f0fdbf49)
Don't look at the particular hue. This will need a lot of work due to transparency - the progress bar is a solid layer _underneath_ the tiers.

**Comment by peppy** [comment, marked as answer]
#23884
